src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
OK.  But I'll update the issue template file on master in a separate PR.  Also, regarding those extra commands to not export.  I'll remove those.  I'm guilty of adding Get-GitBranch for testing purposes but I think Pesters's InModuleScope will get me access to this function.